.github/workflows/pr-labeling-automation.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Pull Request Labeler'
+on:
+    workflow_dispatch:
```

Comment:
`workflow_dispatch` events may not be as useful as we'd like here.

When manually dispatching a workflow run, a branch is selected from the dropdown. Only branches on the current repository are available. This means that this workflow cannot be manually run for any PR submitted from a fork.

Probably makes sense to leave it, but just wanted to point that out.